erpnext/manufacturing/doctype/routing/test_routing.py

```diff
@@ -61,6 +75,7 @@ def create_routing(**args):
 
 	if not args.do_not_save:
 		try:
+			doc.set('operations', {})
```

Comment:
```suggestion
			doc.set('operations', [])
```
Child table fields are a list.